### PR TITLE
Store event id and trigger data as integers, not strings

### PR DESCRIPTION
**Pull request by apasel422**
***
<a href="https://pr-preview.s3.amazonaws.com/apasel422/conversion-measurement-api/pull/266.html" title="Last updated on Nov 5, 2021, 7:00 PM UTC (21a61bd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/266/cabacc3...apasel422:21a61bd.html" title="Last updated on Nov 5, 2021, 7:00 PM UTC (21a61bd)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -813,6 +813,10 @@ To <dfn>serialize an [=attribution report=]</dfn> |report|, run the following st
 
 1. Return the [=byte sequence=] resulting from executing [=serialize an infra value to JSON bytes=] on |data|.
 
+Issue: How can we force [=attribution report/event id=] and [=attribution report/trigger data=] to
```
Comment (johnivdel) [inline index.bs:816]: Could we directly convert these to strings before placing them in the map?

Comment (apasel422) [reply index.bs:816]: I couldn't find a way to convert a 64-bit integer (or any integer, necessarily) into a string, without relying on a JS function.

Comment (johnivdel) [reply index.bs:816]: I think we can copy what fetch does for now: https://fetch.spec.whatwg.org/#serialize-an-integer

File: index.bs [hunk 2]
```diff
@@ -794,6 +794,13 @@ To <dfn>queue a report for delivery</dfn> given an [=attribution report=] |repor
     Note: This is intended to allow user agents to optimize device resource usage.
 1. Run [=attempt to deliver a report=] with |report|.
 
+<h3 id="serialize-integer">Serialize an integer</h3>
+
+To <dfn>serialize an integer</dfn>, represent it as a string of the shortest possible decimal number.
+
+Issue: This will be replaced by a more descriptive algorithm in Infra. See
```
Comment (johnivdel) [inline index.bs:801]: Given that issue has been open a long time, maybe something like "This would ideally use a..."

**Review by johnivdel — APPROVED** [review]

**Pull request merged**